templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -455,6 +462,16 @@ parameters:
   name: BROKER_USER
   value: YWRtaW4=
 
+- description: Dockerhub user password
+  displayname: Dockerhub user password
+  name: DOCKERHUB_PASS
+  value: Y2hhbmdlbWUK
```

Comment:
Is this a real docker user and pw?

Reply:
This is "changeme/changeme" base64 encoded so I figured it was safe. I can make it an empty string too.

Reply:
@dymurray now that we have the ability to do empty username/password I don't see why we wouldn't do that?